ci/publish_docker.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+# Copyright IBM Corp. All Rights Reserved.
+#
+# SPDX-License-Identifier: Apache-2.0
+set -eu -o pipefail
+
+make docker
+docker login --username "${DOCKER_USERNAME}" --password "${DOCKER_PASSWORD}"
+docker tag "hyperledger/fabric-ca" "hyperledger/fabric-ca:amd64-${RELEASE}"
+docker push "hyperledger/fabric-ca:amd64-${RELEASE}"
+
+sudo wget -qO /usr/local/bin/manifest-tool https://github.com/estesp/manifest-tool/releases/download/v1.0.0/manifest-tool-linux-amd64
```

Comment:
why sudo? Do we need to put this into /usr/local/bin?

Reply:
Technically no, but then I'd have to populate the path variable in the variables section, or execute it locally

Reply:
Changed to executing manually